daisy/common.go

```diff
@@ -159,19 +159,36 @@ func (w *Workflow) substituteSourceVars(ctx context.Context, v reflect.Value) DE
 	})
 }
 
+type traverseAction uint
+
+const (
+	continueTraverse traverseAction = iota
+	prune
+)
```

Comment:
These should be documented, the type and the constants.

Reply:
Done!